frontend/src/components/common/IntegerInput.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from "react";
+import {observer} from "mobx-react";
+import PropTypes from "prop-types";
+
+import {randomString} from "../../common";
+
+@observer
+class IntegerInput extends Component {
+    constructor(props) {
+        super(props);
+        this._id = randomString();
+    }
+    render() {
+        const {onChange, label, value} = this.props;
+        return (
+            <div className="form-group">
+                {label ? <label htmlFor={this._id}>{label}</label> : null}
+                <input
+                    id={this._id}
+                    className="form-control"
+                    type="number"
```

Comment:
make `step=1` for an integer input so you can't get floats

Reply:
done